UsersController.cs

```diff
@@ -0,0 +1,9 @@
+[HttpPost("delete/{id}")]
+public void Delete(uint id)
+{
+    User user = _context.Users.FirstOrDefault(user => user.Id == id);
+    _context.Users.Remove(user);
+    _context.SaveChanges();
```

Comment:
If available, we should use async overloads of methods - note that `Remove()` does not have an async overload.
We should also use cancellation token.
Generally application logic - in this example a logic for getting and removing a single user - should be stored in a separate service class and injected into controller or invoked otherwise e.g. using IMediator library and CQRS approach.
Deleting user logic might also be moved to a repository implementation.
In this way we avoid spilling application and infrastructural logic into controller's actions.
Additionally we might want to handle a case when selected user is already deleted (especially when we are using FirstOrDefault which potentially can return a null value for user) e.g. by throwing an exception or returning different response code by controller e.g. 400 or 404.

```suggestion
    User user = await _context.Users.FirstOrDefaultAsync(user => user.Id == id, cancellationToken);
    if (user is null)
    {
      // handle user already deleted or not found
    }
    _context.Users.Remove(user);
    await _context.SaveChangesAsync(cancellationToken);
```